classes/ui.js

```diff
@@ -0,0 +1,25 @@
+import BookCollection from './book-collection.js';
+
+export default class UI {
+  static #bookListElement = document.querySelector('.book-list');
+
+  static #bookElement = document.querySelector('.book');
+
+  static addBook(book) {
+    const card = UI.#bookElement.cloneNode(true);
+    card.querySelector('.title-author').textContent = book.toString();
+    card.querySelector('button').addEventListener('click', (event) => UI.removeBook(event, book));
+    this.#bookListElement.appendChild(card);
+  }
+
+  static removeBook(event, book) {
+    const buttonElement = event.target;
+    const cardElement = buttonElement.parentElement;
+    cardElement.remove();
+    BookCollection.remove(book.id);
+  }
```

Comment:
- [ ] **Required:** You have done an amazing job so far. 👏 
However, your local storage is not working properly. When an item is removed, your entire store is broken. Kindly fix this behaviour.

![remove empties local storage](https://user-images.githubusercontent.com/27709832/164069945-84f6842e-8526-40c9-a154-ff489c6bef13.gif)